runtime/Python3/src/antlr4/TokenStreamRewriter.py

```diff
@@ -112,12 +122,12 @@ def getText(self, program_name, interval):
         if start < 0: start = 0
 
         # if no instructions to execute
-        if not rewrites: return self.tokens.getText(interval)
+        if not rewrites: return self.tokens.getText(Interval(start, stop))
```

Comment:
same here

Reply:
No, it's not the same. The code above does some range check and make sure ``start`` and ``stop`` are  in proper range. If we just pass the original ``interval`` with an out-of-range ``stop``, error will be raised in ``BufferedTokenStream.getText``

Reply:
I mean pass range(start, stop)
I already mentioned I was not keen to include Interval in python 3